### DISCUSSION TITLE
Acceptance criteria for a new command

### DISCUSSION THREAD

**Discussion by noraj** [tldr-pages/tldr#10797] [Q&A] [answered]
What are the acceptance criteria for a new command?

It should be open-source? Spread / popular enough? Available in at least X distribution official repositories?

**Comment by kbdharun** [comment, marked as answer]
> What are the acceptance criteria for a new command?
> 
> It should be open-source? Spread / popular enough? Available in at least X distribution official repositories?

There isn't a specific criteria for adding a tldr page for a command. You can add commands regardless of whether it is open-source/popular/proprietary/a relatively new project. We don't expect the commands to exist in any distribution repository/popular publishing platforms (PyPi, etc.), we can even include commands which require self compilation & installation and are not present in a distribution repository. 

But while creating a page, if a package works cross-platform you can add it to the `common` directory or else you can add it to the respective platform-specific directories. (In some cases, we have commands which work the same on Linux and Mac whereas it might be an alias in Windows (checkout curl) in that instance you can create an additional page under Windows for the command)

Do note in some instances of name collisions (the same name being used by multiple commands) there might be some special rules enforced by the maintainers, we are currently working on a way to structure it like this for example if it's a JS framework we suffix the page name with something like `just.js` and add the conflicting page with something like `just.1` referring both in the common page (`just`). (Ref. https://github.com/tldr-pages/tldr/pull/10783)

**Reply by noraj** [reply]
Thanks! It sounds like you make it the easiest to contribute to tldr ❤️

**Reply by sbrl** [reply]
@noraj that's the idea! We want tldr-pages to be a welcoming place for contributors of all experience levels - there's no way we'd manage to document everything on our own haha - we need all the help we can get! :D
